partiql-logical-planner/src/lower.rs

```diff
@@ -842,21 +974,39 @@ impl<'ast> Visitor<'ast> for AstToLogical {
             Lit::IonStringLit(s) => parse_embedded_ion_str(s),
```

Comment:
You'll need to fix `parse_embedded_ion_str` to not expect and to return a `Result`, I think

Reply:
Resolved in https://github.com/partiql/partiql-lang-rust/pull/366